docs/core/tutorials/nft-tutorials/crc721.md

```diff
@@ -0,0 +1,213 @@
+
+    constructor(
+        string memory name_,
+        string memory symbol_
+    ) ERC721(name_, symbol_) Ownable(msg.sender) {}
+
+    function mint(address to, string memory uri) public onlyOwner {
+        _safeMint(to, nextTokenId);
+        _setTokenURI(nextTokenId, uri);
+        nextTokenId++;
+    }
+}
+
+```
+
+This contract uses OpenZeppelin's ERC721URIStorage and Ownable contracts to create an NFT that only the owner can mint.
+
+## Step 4: Configuring Hardhat
+
+Edit the `hardhat.config.js` to add the Conflux network configuration:
+
+```javascript
+require("@nomicfoundation/hardhat-toolbox");
+require("hardhat-conflux");
+
+const PRIVATE_KEY = "YOUR_PRIVATE_KEY_HERE";
+
+/** @type import('hardhat/config').HardhatUserConfig */
+module.exports = {
+  solidity: "0.8.24",
+  defaultNetwork: "cfxTestnet",
+  networks: {
+    cfx: {
+      url: "https://main.confluxrpc.com",
+      accounts: [PRIVATE_KEY],
+      chainId: 1029,
+    },
+    cfxTestnet: {
+      url: "https://test.confluxrpc.com",
+      accounts: [PRIVATE_KEY],
+      chainId: 1,
+    },
+  },
+};
+```
+
+Replace `YOUR_PRIVATE_KEY_HERE` with your private key.
+
+## Step 5: Deploying the Contract
+
+Create a deployment script in the `scripts` folder, named `deploy.js`:
+
+```javascript
+const hre = require("hardhat");
+
+async function main() {
+  const signers = await hre.conflux.getSigners();
+  const defaultAccount = signers[0];
+
+  const ConfluxCRC721NFT = await hre.conflux.getContractFactory(
+    "ConfluxCRC721NFT"
+  );
+  const receipt = await ConfluxCRC721NFT.constructor("Confi", "Confi")
+    .sendTransaction({
+      from: defaultAccount.address,
+    })
+    .executed();
+
+  console.log(
+    `Contract deployment ${
+      receipt.outcomeStatus === 0 ? "succeeded" : "failed"
+    }`
+  );
+
+  console.log("ConfluxCRC721NFT deployed to:", receipt.contractCreated);
+}
+
+main()
+  .then(() => process.exit(0))
+  .catch((error) => {
+    console.error(error);
+    process.exit(1);
+  });
+```
+
+Deploy the contract by running:
+
+```bash
+npx hardhat run scripts/deploy.js --network cfxTestnet
+```
+
+## Step 6: Minting an NFT
+
+To mint an NFT, you can use a script that interacts directly with the `mint` function in your smart contract. This script will mint an NFT to a specified address.
+
+Create another script in the `scripts` folder and name it `mintNFT.js`:
+
+This script sets the URI of a specific token, where `YOUR_CONTRACT_ADDRESS` is the address of your deployed NFT contract and `NFT_RECEIVER_ADDRESS` is the address of the NFT you want mint to.
+
+```javascript
+const hre = require("hardhat");
+
+async function main() {
+  const signers = await hre.conflux.getSigners();
+  const defaultAccount = signers[0];
+
+  const contractAddress = "YOUR_CONTRACT_ADDRESS";
+  const recipientAddress = "NFT_RECEIVER_ADDRESS"; // Address to receive the NFT
+  const tokenURI = "https://yourdomain.com/metadata/1.json"; // The metadata URI for the NFT
```

Comment:
It is quite hard to configure a token URI from scratch. Would suggest to provide a template toke uri here
```suggestion
  const tokenURI = "https://raw.githubusercontent.com/conflux-fans/dual-space-nft-metadata/main/2023040104"; //  Replace the example tokenURI with the actual metadata URI for the NFT
```